src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -801,11 +811,30 @@ public function testPutState()
         $this->assertEquals(1, $response['nodeState']);
     }
 
+    private function import($file)
+    {
+        if ($this->session->getRootNode()->hasNode('cmf')) {
+            $this->session->getNode('/cmf')->remove();
+            $this->session->save();
+        }
+
+        $this->session->importXML(
+            '/',
+            __DIR__ . '/../../app/Resources/exports/' . $file . '.xml',
+            ImportUUIDBehaviorInterface::IMPORT_UUID_COLLISION_REPLACE_EXISTING
+        );
+        $this->session->save();
+    }
+
     private function buildTree()
     {
-        $data = [
+        $this->import('tree-dump');
```

Comment:
Wouldn't use the suffix `dump` here, since `export` in the path already says what this is about.